packages/i18n/src/locales/en.i18n.json

```diff
@@ -6419,5 +6419,6 @@
   "Close_gallery": "Close gallery",
   "Next_image": "Next Image",
   "Previous_image": "Previous image",
-  "Image_gallery": "Image gallery"
+  "Image_gallery": "Image gallery",
+  "Youre_unavailable": "You’re unavailable"
```

Comment:
```suggestion
  "You_cant_take_chats_unavailable": "You cannot take new conversations because you're unavailable",
  "You_cant_take_chats_offline": "You cannot take new conversations because you're offline"
```